src/content/getusermedia/getdisplaymedia/index.html

```diff
@@ -25,25 +25,26 @@
   <link rel="icon" sizes="192x192" href="../../../images/webrtc-icon-192x192.png">
   <link href="//fonts.googleapis.com/css?family=Roboto:300,400,500,700" rel="stylesheet" type="text/css">
   <link rel="stylesheet" href="../../../css/main.css">
+  <link rel="stylesheet" href="css/styles.css">
+  <script src="https://unpkg.com/@webcomponents/webcomponentsjs/webcomponents-loader.js"></script>
 
 </head>
 
 <body>
 
   <div id="container">
-    <h1><a href="//webrtc.github.io/samples/" title="WebRTC samples homepage">WebRTC samples</a> <span>getDisplayMedia</span></h1>
+    <h1><a href="//webrtc.github.io/samples/" title="WebRTC samples homepage">WebRTC samples</a> <span>Demo of getDisplayMedia and screen recording</span></h1>
 
-    <video id="gum-local" autoplay playsinline></video>
+    <h4>Screen capturing is currently an experimental feature which is onyl supported by latest Chrome and Firefox!</h4>
```

Comment:
Typo and factually incorrect, since Edge was the [first browser to support getDisplayMedia](https://blog.mozilla.org/webrtc/getdisplaymedia-now-available-in-adapter-js/).

Reply:
Thanks. Will fix!